jupyterlite_sphinx/jupyterlite_sphinx.py

```diff
@@ -614,6 +618,22 @@ def jupyterlite_build(app: Sphinx, error):
             jupyterlite_dir,
         ]
 
+        if jupyterlite_build_command_options is not None:
+            for key, value in jupyterlite_build_command_options.items():
+                # Check for conflicting options from the default command we use
+                # while building. We don't want to allow these to be overridden
+                # unless they are explicitly set through Sphinx config.
+                if key in ["contents", "output_dir", "lite_dir"]:
+                    jupyterlite_command_error_message = """
+                Cannot use [contents, output_dir, lite_dir] in jupyterlite_build_command_options.
+                Please use the corresponding option directly in conf.py.
+                For a list of available options, run the 'jupyter lite build --help-all'
+                command and refer to the documentation for jupyterlite-sphinx:
+                https://jupyterlite-sphinx.readthedocs.io/en/stable/configuration.html
+                """
+                    raise ValueError(jupyterlite_command_error_message)
+                command.extend([f"--{key}", value])
```

Comment:
```suggestion
                    raise ValueError(jupyterlite_command_error_message)
                command.extend([f"--{key}", str(value)])
```

I think it's fine just to convert these to `str` here, and then you can leave the `Any` in the type hints.